.ado/ConditionalNugetAdd.ps1

```diff
@@ -0,0 +1,5 @@
+$sources = nuget sources list
+$found? = $sources -Match "https://api.nuget.org/v3/index.json"
```

Comment:
For my education, why is this variable sufficed with a `?`?
Does this make it nullable/optional?
If so, why does it need to be nullable/optional?

Reply:
@JunielKatarn just part of the variable name, old habits die hard I guess.. shouldn't have any effect, but I might rename/get rid of `?` if it's confusing.